### PR TITLE
feat(Header): Add Search Jobs link

### PR DESCRIPTION
**Pull request by AlaaHamidah**
As in `JobStreet` we are not able to provide the search bar for the user inside JDP, so as an alternative solution we will add the `Search Jobs` link to the header in order to help him going back to search page

image

### CODE REVIEW

**Comment by ZhenYongSeekAsia** [comment]
can you add mobile view?

**Comment by AlaaHamidah** [comment]
image
@ZhenYongSeekAsia

File: jobStreet/Header/Header.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const getJobStreetProps = ({ country, language }) => {
   const headerMessage = _get(messages, 'header');
 
   const links = [
+    { ..._get(headerMessage, 'searchJobs'), ItemIcon: SearchIcon },
```
Comment (ZhenYongSeekAsia) [inline jobStreet/Header/Header.js]: where this SearchIcon shown? 🤔

Comment (AlaaHamidah) [reply jobStreet/Header/Header.js]: it's not shown :D but somehow used.
if I remove it will loose the left padding

Comment (AlaaHamidah) [reply jobStreet/Header/Header.js]: I just followed what is currently used in other tabs, to avoid breaking anything.
Let me know if you have concerns

Comment (ashin) [reply jobStreet/Header/Header.js]: I think it used to be seen in mobile. But those icons are invisible now, for some reason.

I'm happy for this to go out as is, and we can add an investigation card to see if this is an issue or not.

**Pull request merged**